recipes/cairomm/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set version = "1.12.0" %}
+{% set checksum = "5a09a3f604cec2ddb4dfcace28dbe74a53585585" %}
+{% set checktype = "sha1" %}
+
+package:
+    name: cairomm
+    version: {{ version }}
+
+source:
+    fn: cairomm-{{ version }}.tar.gz
+    url: https://www.cairographics.org/releases/cairomm-{{ version }}.tar.gz
+    {{ checktype }}: {{ checksum }}
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  # Packages required to build the package. python and numpy must be
+  # listed explicitly if they are required.
+  build:
+    - cairo
+    - libsigcpp
+    - libpng
+    - pkg-config
+    - toolchain
+  run:
+    - cairo
+    - libsigcpp
+    - libpng
+
+test:
+    commands:
+        - test -f $PREFIX/lib/libcairomm-1.0.a           # [linux]
```

Comment:
Static library should be on both platforms right?

Also, please test for the Linux shared library.